src/com/cft/contactmerge/contact/tests/NameTests.java

```diff
@@ -180,4 +180,31 @@ void Name_setParseSuffixAndPrefixInLastName_PunctuatedLastName(){
         assertEquals("Mr.", personOne.getPrefix(), testFailedMessage("Mr.", personOne.getPrefix()));
         assertEquals("Jr.", personOne.getSuffix(), testFailedMessage("Jr.", personOne.getSuffix()));
     }
+
+    @Test
+    void Name_isMiddleNameInFullName_True_PunctuationIgnoreSpaces(){
+        Name name = new Name();
+
+        assertTrue(name.isMiddleNameInFullName("Mr.Page,Christopher,James,1st"));
```

Comment:
Need a verification message on all these if you a going to have more than one assert per test so that we know which case failed. assertTrue and assertFalse provide no information about what was asserted.

Reply:
I will add them.